src/components/app.test.js

```diff
@@ -0,0 +1,42 @@
+// app.test.js
+
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act } from 'react-dom/test-utils';
+
+import { IntlProvider } from 'react-intl';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import App from './app';
+import { store } from '../redux/store';
+
+let container = null;
+beforeEach(() => {
+    // setup a DOM element as a render target
+    container = document.createElement('div');
+    document.body.appendChild(container);
+});
+
+afterEach(() => {
+    // cleanup on exiting
+    unmountComponentAtNode(container);
+    container.remove();
+    container = null;
+});
+
+it('renders', async () => {
+    await act(async () =>
+        render(
+            <IntlProvider locale="en">
+                <Provider store={store}>
+                    <BrowserRouter>
+                        <App />
+                    </BrowserRouter>
+                </Provider>
+            </IntlProvider>,
+            container
+        )
+    );
+
+    expect(container.textContent).toBe('GridXXXAppJohn DoeConnected');
```

Comment:
why GridXXXAppJohn DoeConnected and not GridXXXApp John Doe Connected ??
it's related to eslint pretiier pr ?